src/webmon_app/reporting/templates/dasmon/dashboard.html

```diff
@@ -102,7 +107,25 @@
         </tbody>
     </table></td>
   </table>
+  <script>
```

Comment:
Would this be better going into its own js file in https://github.com/neutrons/data_workflow/tree/next/src/webmon_app/reporting/static since this same code is used in two different places?

Reply:
Ok. I'll do that while fixing the issue below.